cypress/support/pages/courses/pageCommonCourse.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+class Common_Course {
```

Comment:
Sometimes it's Common_XYZ and then in other cases it's XYZ_COMMON...
Not sure what's the prefered way, but it should be the same everywhere!
Please search for COMMON and align based on your decision, there are like 100 occurences and I don't want to leave a comment for all of them.